### DISCUSSION TITLE
List all rules that have violations in the summary output

### DISCUSSION THREAD

**Discussion by misilot** [ansible/ansible-lint#1105] [Ideas] [answered]
For example in this output

```
│ # .ansible-lint                                                                                                                                                                                          │
│ warn_list:  # or 'skip_list' to silence them completely                                                                                                                                                  │
│   - experimental  # all rules tagged as experimental
```
It would be great if it listed all the errors. For example Rule [208] File permissions unset or incorrect does not show up in the listing even when there are multiple errors.

As an aside, it would be also cool if there was a count for the total of each violation.

Thanks!

**Comment by ssbarnea** [comment, marked as answer]
A PR for each of these features would be welcomed.

**Reply by misilot** [reply]
@ssbarnea I will take a look to see why rules when they are matched don't get added to the matches array. I believe that is what isn't happening.
